src/screen/320x240x16/lcd_string.c

```diff
@@ -295,3 +295,6 @@ void LCD_GetStringDimensions(const u8 *str, u16 *width, u16 *height) {
 void LCD_SetFontColor(u16 color) {
     cur_str.color = color;
 }
+
+#define TESTNAME string
+#include "tests.h"
```

Comment:
we probably shouldn't call this test function 'string'.  either 'lcd_string' or 'screen_string' or whatever

Reply:
how about call it text?

Reply:
I'd prefer 'font' or 'drawtext'